<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParaGUI</class>
 <widget class="QWidget" name="ParaGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>1030</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_17">
     <item>
      <widget class="QSpinBox" name="anz_um">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="value">
        <number>6000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="anz_um_label">
       <property name="text">
        <string>Anzahl Umdrehungen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QSpinBox" name="anz_teil">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>15000</number>
       </property>
       <property name="value">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="anz_teil_label">
       <property name="text">
        <string>Anzahl Teilchen pro Thread</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QSpinBox" name="anz_threads">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>8</number>
       </property>
       <property name="value">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="anz_threads_label">
       <property name="text">
        <string>Anzahl Threads</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QSpinBox" name="hist_anz">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>128</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="hist_anz_label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Anzahl Histogrammbalken&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QDoubleSpinBox" name="nukl_ener">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>11.400000000000000</double>
       </property>
       <property name="maximum">
        <double>120.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>11.400000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="nukl_ener_label">
       <property name="text">
        <string>Initial Nukleonenenergie</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QSpinBox" name="save_Gap">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="save_Gap_label">
       <property name="text">
        <string>Save Gap</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QComboBox" name="spannung">
       <item>
        <property name="text">
         <string>einfachharmonisch</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>doppeltharmonisch</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="spannung_label">
       <property name="text">
        <string>Spannungstyp</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QComboBox" name="ionen_sorte">
       <item>
        <property name="text">
         <string>Uran 73+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Uran 28+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Argon 18+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Proton 1+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Neon 10+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nickel 26+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>86Krypton 33+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>86Krypton 34+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bismut 67+</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ionen_sorte_label">
       <property name="text">
        <string>Ionensorte</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QComboBox" name="regler">
       <item>
        <property name="text">
         <string>kein Regler</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3_TAP_Filter_EXP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3_TAP_Filter_Liwei</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3_TAP_Filter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Zustandsregler</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Energy_Sink_Regler</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hamilton_Damp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hamilton_Damp_2</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="regler_label">
       <property name="text">
        <string>Regler</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="coasting">
     <property name="text">
      <string>Coasting Beam    |     FÃ¼llgrad</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="fuellgrad">
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="beschleunigung">
     <property name="text">
      <string>Beschleunigung</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDoubleSpinBox" name="noise_power">
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>3.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="noise">
       <property name="text">
        <string>Messrauschen %</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QDoubleSpinBox" name="FIR_gain">
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>0.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.000100000000000</double>
       </property>
       <property name="value">
        <double>-0.003000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="gain_label">
       <property name="text">
        <string>Gain</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QSpinBox" name="u_1_spinbox">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>60000</number>
       </property>
       <property name="value">
        <number>4000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="u_1_label">
       <property name="text">
        <string>Grundspannung</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QSpinBox" name="harmonic_number">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="harmonic_number_label">
       <property name="text">
        <string>h</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QDoubleSpinBox" name="frequenz_modifier">
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="minimum">
        <double>0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>2.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.100000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="frequenz_modifier_label">
       <property name="text">
        <string>frequenz_modifier</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="single_particle">
     <property name="text">
      <string>Single Particle</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>StÃ¶rung</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QSpinBox" name="distortion_angle">
          <property name="maximum">
           <number>40</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="distortion_angle_label">
          <property name="text">
           <string>Auslenkung Winkel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QSpinBox" name="distortion_steps_coasting">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="distortion_steps_coasting_label">
          <property name="text">
           <string>Steps nach Coasting</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="distortion_active">
        <property name="text">
         <string>StÃ¶rung aktiv</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QSpinBox" name="distortion_time_repeat">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>1000000</number>
          </property>
          <property name="value">
           <number>2500</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="distortion_repeat_active">
          <property name="text">
           <string>Wiederholen (Âµs)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
